src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1575,6 +1575,40 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
         [Theory]
         [InlineData(false)]
         [InlineData(true)]
+        [InlineData(null)]
+        public async Task PostAsync_ExpectContinue_Success(bool? expectContinue)
```

Comment:
What is this test supposed to do on UWP and .NET Core (not managed handler)?  On .NET Framework, expect-100-continue is supported also.